pallets/stateful-storage/Cargo.toml

```diff
@@ -32,26 +32,11 @@ sp-std = { git = "https://github.com/paritytech/substrate", default-features = f
 common-primitives = { default-features = false, path = "../../common/primitives" }
 
 [dev-dependencies]
+env_logger = "0.10.0"
 pretty_assertions = "1"
 
 [features]
 default = ['std']
-runtime-benchmarks = [
-  "frame-benchmarking/runtime-benchmarks",
-  "frame-support/runtime-benchmarks",
-  "frame-system/runtime-benchmarks",
-  "common-primitives/runtime-benchmarks",
-]
-std = [
-  'codec/std',
-  'scale-info/std',
-  'sp-std/std',
-  'sp-core/std',
-  'sp-io/std',
-  'sp-runtime/std',
-  'frame-support/std',
-  'frame-system/std',
-  'frame-benchmarking/std',
-  'common-primitives/std',
-]
+runtime-benchmarks = ["frame-benchmarking/runtime-benchmarks", "frame-support/runtime-benchmarks", "frame-system/runtime-benchmarks", "common-primitives/runtime-benchmarks"]
```

Comment:
We should use the same 1 item per line styling, It's easier to see the diffs

Reply:
Weird; must have been some auto-formatter hook. Fixed.